go/vt/servenv/grpc_server.go

```diff
@@ -127,7 +127,7 @@ var (
 func RegisterGRPCServerFlags() {
 	OnParse(func(fs *pflag.FlagSet) {
 		fs.IntVar(&gRPCPort, "grpc_port", gRPCPort, "Port to listen on for gRPC calls. If zero, do not listen.")
-		fs.StringVar(&gRPCBindAddress, "grpc_bind_address", gRPCBindAddress, "Bind address for gRPC calls. If empty, listen on all addresses.")
+		fs.StringVar(&gRPCBindAddress, "grpc_bind-address", gRPCBindAddress, "Bind address for gRPC calls. If empty, listen on all addresses.")
```

Comment:
should we go full-dash here? (i know the other flags use `grpc_` as a prefix, we can also "normalize": https://github.com/spf13/pflag#mutating-or-normalizing-flag-names)

Reply:
I reverted it to use only `_`, we can find a longer term fix for all flags later.